### PR TITLE
Added Functions & Threshold plugin digest prefixes

### PR DESCRIPTION
**Pull request by KuphJr**
Since each OCR plugin must have a unique config digest, in order to support multiple OCR plugins with a single OCR2Base contract, different prefixes must be used for the configDigest to ensure that each configDigest is unique even though the underlying config data is identical.

These 2 additional configDigest prefixes will support the Threshold decryption plugin and S4 storage plugin for Chainlink Functions.

### CODE REVIEW

**Comment by KuphJr** [comment]
Since each OCR plugin must have a unique config digest, in order to support multiple OCR plugins with a single OCR2Base contract, different prefixes must be used for the configDigest to ensure that each configDigest is unique even though the underlying config data is identical.

These 2 additional configDigest prefixes will support the Threshold decryption plugin and S4 storage plugin for Chainlink Functions.

File: offchainreporting2plus/types/config_digest.go [hunk 1]
```diff
@@ -21,6 +21,8 @@ const (
 	ConfigDigestPrefixSolana     ConfigDigestPrefix = 3
 	ConfigDigestPrefixStarknet   ConfigDigestPrefix = 4
 	ConfigDigestPrefixMercuryV02 ConfigDigestPrefix = 6
+	ConfigDigestPrefixThreshold  ConfigDigestPrefix = 7
```
Comment (bolekk) [inline offchainreporting2plus/types/config_digest.go:24]: Can you add a comment here as well: "prefixes for running Threshold/S4 plugins as part of another product under  one contract."

Comment (bolekk) [reply offchainreporting2plus/types/config_digest.go:24]: Another option is to reserve a range for any "secondary" plugin...

**Comment by KuphJr** [comment]
@kaleofduty I don't seem to be able to tag you for review here for some reason, but can we get this merged to ensure no one else in the future tries to re-use these prefixes which have been reserved for Functions?

**Comment by kaleofduty** [comment]
this has been addressed

**Pull request closed without merging**